profiles/base-empty.yaml

```diff
@@ -53,6 +53,7 @@ empty_values:
   classes_to_teach: []
   whirlwind_trainables: []
   aiming_trainables: []
+  attack_overrides: []
```

Comment:
❓ Shouldn't this be a hash (and not an array)?

Reply:
Fixed!